src/generators/subcomponent.ts

```diff
@@ -13,30 +13,31 @@ export const createSubcomponent = async ({
   componentPath,
 }: IOptions) => {
   let componentsTemplatePath;
-  const extension = isOnlyJs ? "js" : "ts";
+  const extension = isOnlyJs ? 'js' : 'ts';
+  const componentName = 'Test';
   if (isOnlyJs)
-    componentsTemplatePath = `${PATH.SUBCOMPONENT_TEMPLATE}/javascript/button.hbs`;
+    componentsTemplatePath = `${PATH.SUBCOMPONENT_TEMPLATE}/javascript/test.hbs`;
   else
-    componentsTemplatePath = `${PATH.SUBCOMPONENT_TEMPLATE}/typescript/button.hbs`;
+    componentsTemplatePath = `${PATH.SUBCOMPONENT_TEMPLATE}/typescript/test.hbs`;
 
   const componentsPath = `${componentPath}/components`;
-  const componentsFolderPath = `${componentsPath}/Button`;
+  const componentsFolderPath = `${componentsPath}/Test`;
```

Comment:
usemos component name ya que es lo mismo 
```suggestion
  const componentsFolderPath = `${componentsPath}/${componentName}`;
```